src/server/pachyderm_test.go

```diff
@@ -6317,6 +6317,87 @@ func TestGetFileWithEmptyCommits(t *testing.T) {
 	require.True(t, strings.Contains(err.Error(), "not found"))
 }
 
+// TestDeleteCommitPropagation deletes an input commit and makes sure all
+// downstream commits are also deleted.
+// DAG in this test: repo -> pipeline[0] -> pipeline[1]
+func TestDeleteCommitPropagation(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	defer require.NoError(t, c.DeleteAll())
+
+	// Create an input repo
+	repo := uniqueString("TestDeleteCommitPropagation")
+	require.NoError(t, c.CreateRepo(repo))
+
+	pipeline := make([]string, 2)
```

Comment:
I'd make this `2` a constant `numPipelines`

Reply:
Done